client/my-sites/plugins/main.jsx

```diff
@@ -426,7 +425,7 @@ const PluginsMain = React.createClass( {
 export default connect(
 	state => {
 		return {
-			wporgPlugins: state.plugins.wporg
+			plugins: state.plugins.wporg
```

Comment:
Thanks! I'll change the name to reflect that. How do you feel about the following data structure? My main goal was primarily to full the isFetching state out of the plugin object.
```
{ plugins: 
      { wporg: 
             {  items: {},
                fetchingItems: {} 
             }
      }
}


```